lego/apps/events/tests/test_events_api.py

```diff
@@ -310,28 +334,6 @@ def test_creator_see_registrations(self):
         for pool in event_response.json()["pools"]:
             self.assertIsNotNone(pool.get("registrations", None))
 
-    def test_without_auth_permission_abakom_only(self):
```

Comment:
Did we have two copies of the same test?

Reply:
Looks like it yeah